### PR TITLE
Add BSTN token metadata

### PR DESCRIPTION
**Pull request by pysrbastion**
# Add BSTN token metadata

ERC-20: https://etherscan.io/address/0x059a1f1dea1020297588c316ffc30a58a1a0d4a2
NEP-141: https://explorer.mainnet.near.org/accounts/059a1f1dea1020297588c316ffc30a58a1a0d4a2.factory.bridge.near
Aurora ERC-20: https://explorer.mainnet.aurora.dev/address/0x9f1F933C660a1DC856F0E0Fe058435879c5CCEf0

### CODE REVIEW

File: tokens/bstn.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...5nnJw4UNBJhMlnH5%2bynh61XHiTIC3Fi82r54Vjp7Fw5JRdTme/t57z1ghHWdnl/h5bsNVz54T7nqK9hlDISTwUdNIQJs6lgyfu6tl%2bzFrvjzpCr7bbz0dP%2b/t72%2bxjZ%2bea9gIo61mJr//eeq0E2HsolCjbz5Zl7b33hJpinTQUcrI0FHSyLBR2QtKEQk6ag4JOmobCTkiSUMjJyqCgkxVjC7udES%2bnQAhpMRRysjoo6GTVuMlzD1rCfi8IIZcJhZxcHhR0ctmwjp2Qy0DVkUvxNQo5uVwo6KRl%2bMIOcTfj7IQsCTu7kZZDQSdrAhPoCIlDTKMmv4au2lcp5KTVUNDJmsI4OyEKxsfJ2kNBJ4lAdzxpQ%2bhWJ4lCQSeJw7I3UmxojZN0oKCT1PCXb5WWuNNqJ/mG1jhJHQo6yQR2rF3WleWuVnobBiHZZwZCfg2y9AStcZIFKOgkU0y%2b%2bOgwFsv30iVPsovlUpeWkHfUnqA1TrIEBZ1kFieRrjxFcSfpY4l4TX6L5WYky1DQSS6guJPkoYiTfEFBJ7mD4k7WiBm38QtFnOQSCjrJNXbMvVaZQr1%2bL2vcySqYgSg9YU0Mv4VydTdFnOQZCjopFHa2fL0%2bhVLpblrvJB5VJ15/EijtZnY6KRIUdFJYaL0TG4kDKFkWeN1ypzMznRQYCjppG/zYu225U%2bALiy3gpd3WRO5J1Dp2T7/9nx0AIW0ABZ20LbbAd1QmbQtelG6ylGASJIdYlrewxFtZ4BRw0sZQ0AlxsV30VVvgVQz%2bJkhL4GnFZwtlfQs7E/1JlK3nSnWaLnRCHCjohCyB02%2b%2bss0XeSiBpyWfCLp4V8QBZqETsjQUdEJWiG/J1yxhr8htkL67nj3oV4Mn3KJ%2b0HabozyNjoUDFG9CVgYFnZAW4Qu9sIS9Vp90Lfphij080Z5xLO7aQev5AIWbkNZCQSckISaf%2bRNL7OvDVmx%2bm/0ol6%2b0nx3Rd59ziCfWcJ89wS5Zrxcr0%2bi%2bNEPRJmTtoaATkiGc1eY6t9nCryz9uhx2RR/2BEDhvXdQnwcTAWFPCpqdGMxYYmwKrW05u585Ig1boPXP1HO1Yr9mRjkh2eH/B5J0iASY20/2AAAAAElFTkSuQmCC'/%3e %3c/defs%3e %3c/svg%3e",
```
Comment (sept-en) [inline tokens/bstn.json:7]: Please, make sure it contains 4 spaces

File: tokens/bstn.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (sept-en) [inline tokens/bstn.svg:1]: The icon is too big. Please reduce it to 5-7 kB at max.

**Review by sept-en — CHANGES_REQUESTED** [review]

**Comment by pysrbastion** [comment]
@sept-en fixed

**Review by sept-en — COMMENTED** [review]
Approved.

**Pull request merged**